edb/lib/ext/auth.edgeql

```diff
@@ -30,6 +30,19 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
         create constraint exclusive on ((.iss, .sub))
     };
 
+    create type ext::auth::LocalIdentity {
```

Comment:
Not extending `Identity` on purpose?

Reply:
Oops! Nope, an oversight.

Also, how do I provide a constant override for the `iss` property from `Identity`? Maybe it should be readonly as well?

Reply:
I couldn't figure out how to get the default of "local" or a constant "local", so we can just follow up with that on a separate PR.